appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -567,6 +568,14 @@ private void setANameProperty(String aname) {
           SettingsConstants.YOUNG_ANDROID_SETTINGS_APP_NAME, aname);
     }
   }
+  
+  private void setTitleVisibleProperty(String visible) {
```

Comment:
This should be a Boolean instead of a String.

Reply:
@conorshipp actually no, for an example, check scrollable property. Plus onPropertyChange event returns String newValue, so no, I can't do that.